src/utils/getCalendarDaySettings.js

```diff
@@ -10,51 +28,34 @@ export default function getCalendarDaySettings(day, ariaLabelFormat, daySize, mo
     dateIsSelectedAsEndDate,
   } = phrases;
 
-  const daySizeStyles = {
-    width: daySize,
-    height: daySize - 1,
+  const formattedDate = {
+    date: day.format(ariaLabelFormat),
   };
 
-  const useDefaultCursor = (
-    modifiers.has('blocked-minimum-nights')
-    || modifiers.has('blocked-calendar')
-    || modifiers.has('blocked-out-of-range')
-  );
-
-  const selected = (
-    modifiers.has('selected')
-    || modifiers.has('selected-start')
-    || modifiers.has('selected-end')
-  );
-
-  const hoveredSpan = !selected && (
-    modifiers.has('hovered-span')
-    || modifiers.has('after-hovered-start')
-  );
-
-  const isOutsideRange = modifiers.has('blocked-out-of-range');
-
-  const formattedDate = { date: day.format(ariaLabelFormat) };
-
-  let ariaLabel = getPhrase(chooseAvailableDate, formattedDate);
-  if (selected) {
-    if (modifiers.has('selected-start') && dateIsSelectedAsStartDate) {
-      ariaLabel = getPhrase(dateIsSelectedAsStartDate, formattedDate);
-    } else if (modifiers.has('selected-end') && dateIsSelectedAsEndDate) {
-      ariaLabel = getPhrase(dateIsSelectedAsEndDate, formattedDate);
-    } else {
-      ariaLabel = getPhrase(dateIsSelected, formattedDate);
-    }
-  } else if (modifiers.has(BLOCKED_MODIFIER)) {
-    ariaLabel = getPhrase(dateIsUnavailable, formattedDate);
+  if (modifiers.has('selected-start') && dateIsSelectedAsStartDate) {
+    return getPhrase(dateIsSelectedAsStartDate, formattedDate);
+  } if (modifiers.has('selected-end') && dateIsSelectedAsEndDate) {
```

Comment:
I think you're missing the `else` part of this logic

Reply:
oh wait I just noticed that you're returning so this ends up being technically ok, but visually confusing (because it looks like an else if but it's actually a standalone if statement

Reply:
whoops, that'd be great to clean up in a followup :-) @monokrome

Reply:
@backwardok hmm... Think I should use an `else`? Feels bad to use `let` when we don't need it, but also feels worse to have a final return in a branch instead of in the function's root block

Reply:
It shouldn't use an else; but a newline should be inserted between `}` and `if`.